.github/workflows/ci-lint.yml

```
@@ -43,13 +43,31 @@ jobs:
       - name: Install gitlint
         run: |
           python -m pip install gitlint
+      - name: Install towncrier
+        run: |
+          python -m pip install https://github.com/oasisprotocol/towncrier/archive/oasis-master.tar.gz
+      - name: Check for presence of a Change Log fragment (only pull requests)
+        # NOTE: The pull request' base branch needs to be fetched so towncrier
+        # is able to compare the current branch with the base branch.
+        # Source: https://github.com/actions/checkout/#fetch-all-branches.
+        run: |
+          git fetch --no-tags origin +refs/heads/"${BASE_BRANCH}":refs/remotes/origin/"${BASE_BRANCH}"
```

Please, check if this is OK or it should be:
```suggestion
          git fetch --no-tags origin "+refs/heads/${BASE_BRANCH}:refs/remotes/origin/${BASE_BRANCH}"
```